Backend/dev/migrations/dynamic-offer-driver-app/0309-Intelligent-pooling-numRide-changes.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE atlas_driver_offer_bpp.driver_intelligent_pool_config ADD COLUMN num_rides_weightage INT DEFAULT 2;
+ALTER TABLE atlas_driver_offer_bpp.driver_intelligent_pool_config ADD COLUMN max_num_rides INT DEFAULT 336;
+ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ADD COLUMN ride_frequency_score DOUBLE PRECISION;
+ALTER TABLE atlas_driver_offer_bpp.driver_intelligent_pool_config RENAME COLUMN cancellation_ratio_window_option TO cancellation_and_ride_frequency_ratio_window_option;
```

Comment:
don't rename, write query to add new column with copying data from old column and another delete column query with a comment saying - to be run after release is stabilised

Reply:
but this table is not enabled for kv in master